.github/workflows/pr-conventions.yaml

```diff
@@ -0,0 +1,34 @@
+name: Pull Request Conventions
```

Comment:
Following the same convention let's call it `Repo - Pull Request Conventions`

Reply:
done